src/Test.FunctionalTests.Sarif/TestData/Multitool/BaselineOption/ExpectedOutputs/TEST1002.ValidateBaseline.NoResults.sarif

```diff
@@ -222,6 +222,7 @@
         {
           "ruleId": "SARIF2005",
           "ruleIndex": 2,
+          "level": "warning",
```

Comment:
>"level": "warning", [](http://example.com/codeflow?start=10&length=19)

the current behavior is, 
if explicitly set in the code 
".Level=FailureLevel.Warning" 
then this line will be written to Sarif. 
If not explicitly set, this line will not be written.

Reply:
got some input from Michael, let me set this to draft and work on it first